QUEST_JOBSTEP.tsv

```diff
@@ -1402,12 +1402,12 @@ QUEST_JOBSTEP_20150317_001401	Defeat the body of evil spirit
 QUEST_JOBSTEP_20150317_001402	Talk to Wizard Master
 QUEST_JOBSTEP_20150317_001403	Go to Wizard Master in Klaipeda
 QUEST_JOBSTEP_20150317_001404	Collect the materials for Wizard Master
-QUEST_JOBSTEP_20150317_001405	Collect the Skins of Grolls in Spine Heart Dale, Feelers of Chafperors in Fallen Worry Forest, and the nucleuses of Yukopuses from Siaulai Mine Village.
+QUEST_JOBSTEP_20150317_001405	Collect the Skins of Grolls in Spine Heart Dale, Feelers of Chafperors in the Fallen Worry forest, and the nucleuses of Yukopuses from Siaulai Mine Village.
 QUEST_JOBSTEP_20150317_001406	Give the materials to Wizard Master
 QUEST_JOBSTEP_20150317_001407	Collected the materials for Wizard Master. Give it to him.
 QUEST_JOBSTEP_20150317_001408	Collect %s from Spine Heart Dale
-QUEST_JOBSTEP_20150317_001409	Collect %s from Fallen Worry Forest
-QUEST_JOBSTEP_20150317_001410	Collect %s from Mine Village
+QUEST_JOBSTEP_20150317_001409	Collect %s from the Fallen Worry forest
+QUEST_JOBSTEP_20150317_001410	Collect %s from Siaulai Mine Village
```

Comment:
Remove Siaulai and change it to Miners' Village. It is true that in another sentence it says Siaulai Miners' Village. Inconsistency on IMC's part?